lib/screens/home/screens/calendar_screen.dart

```diff
@@ -19,6 +20,7 @@ class CalendarScreen extends ConsumerWidget {
     final strings = ref.watch(selectedStringsProvider);
     final contestTime = ref.watch(contestTimeProvider);
     final languageCode = ref.watch(languageProvider);
+    final registrationLink = ref.watch(configProvider);
```

Comment:
I would change it to config because more values than just registration link can be pulled from it